templates/jarvice-kns-scheduler.yaml

```diff
@@ -18,6 +18,65 @@ roleRef:
   name: cluster-admin
   apiGroup: rbac.authorization.k8s.io
 ---
+{{- if .Values.jarvice_kns_scheduler.https.enabled }}
+{{- if .Values.jarvice_kns_scheduler.https.letsencrypt }}
+apiVersion: v1
+data:
+  credentials.json: {{ .Values.jarvice_kns_scheduler.https.letsencrypt_credentials_json_base64 }}
+kind: Secret
+metadata:
+  name: dns-svc-account
+  namespace: cert-manager
+type: Opaque
+---
+apiVersion: cert-manager.io/v1
+kind: ClusterIssuer
+metadata:
+  name: letsencrypt-kns
+  namespace: jarvice-system
+spec:
+  acme:
+    email: {{ .Values.jarvice_kns_scheduler.https.letsencrypt_user_email }}
+    preferredChain: ""
+    privateKeySecretRef:
+      name: letsencrypt-prod
+    server: https://acme-v02.api.letsencrypt.org/directory
+    solvers:
+    - http01:
+        ingress: {}
+    - dns01:
+        cloudDNS:
+          serviceAccountSecretRef:
+            name: dns-svc-account
+            key: credentials.json
+          project: {{ .Values.jarvice_kns_scheduler.https.letsencrypt_project }}
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: {{ .Values.jarvice_kns_scheduler.https.certificate_name }}
+  namespace: jarvice-system
+spec:
+  secretName: kns-wildcard
+  issuerRef:
+    name: letsencrypt-kns
+    kind: ClusterIssuer
+  dnsNames:
+  - {{ .Values.jarvice_kns_scheduler.https.letsencrypt_requested_wildcard }}
+---
+{{- end }}
+{{- if eq .Values.jarvice_kns_scheduler.https.ingressClassName "traefik" }}
+apiVersion: traefik.containo.us/v1alpha1
+kind: TLSStore
+metadata:
+  name: default
+  namespace: jarvice-system
+spec:
+  defaultCertificate:
+    secretName: {{ .Values.jarvice_kns_scheduler.https.certificate_name }}
+---
```

Comment:
what object(s) are needed to enable `nginx`?

Reply:
I will do that in another PR :)